wiki/List_of_Guides/en.md

```diff
@@ -46,7 +46,7 @@ Mapping Techniques
 ------------------
 
 -   [Mapping Techniques](/wiki/Mapping_Techniques)
--   [Making Good Sliders](/wiki/Mapping_Techniques/Making_Good_Sliders)
+-   [Making Good Sliders](/wiki/Mapping_Techniques/Making_Good_Slider)
```

Comment:
I corrected the link to match the folder name; should I do it the other way around for this one? Somewhere along the way this folder lost its `s`, I guess.